force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -0,0 +1,41 @@
+trigger OpportunityTrigger on Opportunity (before update, before delete) {
+    
+    switch on Trigger.operationType {
+        when BEFORE_UPDATE {
```

Comment:
This sort of thing will help you in assignment 8 when we do trigger frameworks. Just also be sure to know the Trigger.isBefore and Trigger.isUpdate context variables